dlfcn_test.go

```diff
@@ -14,10 +14,18 @@ import (
 	"runtime"
 	"strings"
 	"testing"
+	_ "unsafe" // for go:linkname
```

Comment:
I don't see the go linkname

Reply:
It's an artifact from a previous version, removing it
```suggestion
```